modules/media/ajax/FileUpload.php

```diff
@@ -130,6 +130,7 @@ function uploadFile()
     checkDateTaken($dateTaken);
 
     $fileName  = preg_replace('/\s/', '_', $_FILES["file"]["name"]);
+    $fileName  = urldecode($fileName);
```

Comment:
```suggestion
    // urldecode() necessary to decode double quotes encoded automatically 
    // by chrome browsers to avoid XSS attacks
    $fileName  = urldecode($fileName);
```

Reply:
Ill add the comment when I add the script